compiler/src/iree/compiler/Codegen/LLVMCPU/TileSizeSelection.cpp

```diff
@@ -23,17 +23,21 @@ TilingConfig::TilingConfig(IREE::Codegen::LoweringConfigAttr lc)
   // their corresponding incremental levels. We currently support the following
   // scenarios:
   //   1. [[distribution]]
-  //   2. [[distribution], [vector-parallel]]
-  //   3. [[distribution], [vector-parallel], [vector-reduction]]
+  //   2. [[distribution], [vector-common-parallel]]
+  //   3. [[distribution], [vector-common-parallel], [vector-reduction],
+  //       [vector-inner-parallel]]
```

Comment:
I would like to better understand what this inner parallel level represents. Do you have an example?